src/ray/util/logging.cc

```diff
@@ -173,7 +173,12 @@ void RayLog::UninstallSignalAction() {
   RAY_LOG(DEBUG) << "Uninstall signal handlers.";
   // This signal list comes from glog's signalhandler.cc.
   // https://github.com/google/glog/blob/master/src/signalhandler.cc#L58-L70
-  static std::vector<int> installed_signals({SIGSEGV, SIGILL, SIGFPE, SIGABRT, SIGTERM});
+  std::vector<int> installed_signals({SIGSEGV, SIGILL, SIGFPE, SIGABRT, SIGTERM});
+#ifdef _WIN32  // Do NOT use WIN32 (without the underscore); we want _WIN32 here
+  for (int signal_num : installed_signals) {
+    signal(signal_num, SIG_DFL);
```

Comment:
Can you add a comment on what this does?

Reply:
I just copied it over from Arrow's `cpp/src/arrow/util/logging.cc`.